diff --git a/node_modules/chrome-aws-lambda/build/index.js b/node_modules/chrome-aws-lambda/build/index.js
index a4d3c3d..373a758 100644
--- a/node_modules/chrome-aws-lambda/build/index.js
+++ b/node_modules/chrome-aws-lambda/build/index.js
@@ -21,7 +21,7 @@
   font: async (input) => {
     return (0, util_1.getFont)(input);
   },
-  get puppeteer() {
+  get puppeteer () {
     if (process.env.AWS_EXECUTION_ENV !== undefined) {
       return require(path.join(process.cwd(), 'node_modules', 'puppeteer'));
     }
